sc-kpm/CMakeLists.txt

```diff
@@ -20,8 +20,7 @@ add_subdirectory(sc-agents-common)
 add_custom_target(sc-kpm)
 
 if (${SC_KPM_SCP})
```

Comment:
We don't know how new version of scp-interpreter will be included to sc-machine. Probably, it will be separate repo or it will be not part of sc-kpm. I propose remove SC_KPM_SCP flag on this stage